monitoring/explores/overall_ping_counts.explore.lkml

```diff
@@ -0,0 +1,6 @@
+include: "../views//overall_ping_counts.view.lkml"
+
+explore: overall_ping_counts {
+  view_name: overall_ping_counts
+  hidden:  yes
+}
```

Comment:
the we can remove the two explores here and it should be good to go

Reply:
would I have to rebuild the merged explore?

Reply:
yes